collections/_posts/2023-03-08-Variationnal_Dropout_and_the_local_reparameterization_trick.md

```diff
@@ -0,0 +1,243 @@
+---
+layout: review
+title: "Variational Dropout and the Local Reparameterization Trick"
+tags: Bayesian Inference, Distribution, Dropout, 
```

Comment:
Format tags (because tags should not include space and are simply separated by white space):
```suggestion
tags: bayesian-inference distribution dropout
```